README.md

```diff
@@ -1,6 +1,6 @@
-# Bepaid Elixir Library
```

Comment:
@MikeAndrianov lets use "Bepaid" as a library name. Though gateway service name is "bePaid", I prefer to keep it simple.